Documentation/MigrateToComposer/MigrationSteps.rst

```diff
@@ -252,14 +299,26 @@ additional lines added to the :file:`composer.json` from above:
          }
       }
 
-   There is no harm in that, but it won't have any effect.
+   There is no harm in having that, but it won't have any effect.
```

Comment:
cms-package-dir above is outdated.

See: https://github.com/TYPO3/CmsComposerInstallers